Cargo.toml

```diff
@@ -15,6 +15,9 @@ repository = "https://github.com/alphal00p/gammaloop"
 inherits = "dev"
 opt-level = 2
 
+[profile.release]
+#lto=true
```

Comment:
@benruijl @lcnbr Should we enable lto for release?

Reply:
Ah I was just messing around to see if it improved performance.

Reply:
Yes, LTO may improve performance. `LTO="thin"` can also be tried

Reply:
I didn't benchmark it, but just looking at the gamma chain timing, it seems to make it slightly faster. Maybe 10% even. It does make compilation time go through the roof. "Thin" would be a middle ground.